applications/luci-app-openvpn/luasrc/model/cbi/openvpn-file.lua

```diff
@@ -0,0 +1,61 @@
+-- Licensed to the public under the Apache License 2.0.
+
+local ip       = require("luci.ip")
+local fs       = require("nixio.fs")
+local util     = require("luci.util")
+local uci      = require("luci.model.uci").cursor()
+local cfg_file = uci:get("openvpn", arg[1], "config")
+
+local m = Map("openvpn")
+
+if cfg_file then
+	local p = m:section( SimpleSection )
+	p.template = "openvpn/pageswitch"
+	p.mode     = "file"
+	p.instance = arg[1]
+
+	if not fs.access(cfg_file) then
+		local f = SimpleForm("error", nil, translatef("The OVPN config file (%s) could not be found, please check your configuration.", cfg_file))
+		f.reset = false
+		f.submit = false
+		return m, f
+	end
+
+	if fs.stat(cfg_file).size >= 102400 then
```

Comment:
Do you think 100KB is enough? Not sure how large a .conf with embedded cert chains and the like can become

Reply:
I thought 100KB is the limit for Simpleform!? This limit applies only to the Online-Editor, not for the upload of the ovpn file itself.

Reply:
Yeah, but it could be raised if needed, hence the question.